lib/3scale_toolbox/commands/import_command/openapi/method.rb

```diff
@@ -16,7 +16,7 @@ def friendly_name
           end
 
           def system_name
-            friendly_name.downcase
+            friendly_name.downcase.tr(' ', '_')
```

Comment:
Shouldn't be easier make this a whitelist instead of a blacklist?
Like in the other PR, check for `/[^\w]/` ?

Reply:
:+1: